apps/backend/src/modules/users/user.controller.ts

```diff
@@ -38,6 +41,20 @@ class UserController extends BaseController {
 			method: HTTPRequestMethod.GET,
 			path: UsersApiPath.ROOT,
 		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.update(
+					options as APIBodyOptions<UserUpdateRequestDto> & {
+						user?: { id: number };
```

Comment:
As far as I remember we already have a type for `{ id: number }` in the app

Reply:
Check it and if we already have such type replace every `{ id: number }` in your PR